Crash if the Wi-Fi network is not connected during the start of 'spoofdpi'

### Version

1.4.0 64b5784 (homebrew)

### Country

Belarus

### Operating System

macOS 26.4.1

### How are you running spoofdpi?

If you run `spoofdpi` without any options, when the Wi-Fi network is not connected, it starts and crashes immediately after `udp idle timeout; value=25000ms`, at the stage of starting the server on 127.0.0.1:8080.

### Description

Terminal output:
```
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                         f filter  R clear  ^C quit
                                                                       [signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x1010de8a0]

                                                                                                                                                 goroutine 1 [running]:
                                                                                                                                                                       main.runApp({0x101a2fbb8?, 0x3b442c81d70?}, {0x0, 0x0}, 0x3b442dbc030)
                                   	github.com/xvzc/spoofdpi/cmd/spoofdpi/main.go:165 +0xb80
                                                                                                github.com/xvzc/spoofdpi/internal/config.CreateCommand.func25({0x101a2fbb8, 0x3b442c81d70}, 0x3b442ca8dc8)
                                                                                                                                                                                                         github.com/xvzc/spoofdpi/internal/config/cli.go:463 +0x4a0
                                                         github.com/urfave/cli/v3.(*Command).run(0x3b442ca8dc8, {0x101a2fbb8, 0x3b442c81d70}, {0x3b442bb2030, 0x1, 0x1})
                                                                                                                                                                        	github.com/urfave/cli/v3@v3.6.1/command_run.go:354 +0x1f48
                                github.com/urfave/cli/v3.(*Command).Run(0x101a1c4a8?, {0x101a2fb80?, 0x102095280?}, {0x3b442bb2030?, 0x7?, 0x101195aa0?})
                                                                                                                                                         	github.com/urfave/cli/v3@v3.6.1/command_run.go:94 +0x28
             main.main()
                        	github.com/xvzc/spoofdpi/cmd/spoofdpi/main.go:68 +0x70
```

--- a/cmd/spoofdpi/main.go
+++ b/cmd/spoofdpi/main.go
@@ -124,6 +124,7 @@ func runApp(mainctx context.Context, configDir string, cfg *config.Config) error
 	srv, err := createServer(appctx, logger, cfg, resolver)
 	if err != nil {
 		logger.Error().Err(err).Msg("failed to create server")
+		return err
 	}
 
 	logger.Info().Msg("dns info")

